### PR TITLE
Add host networking to hm-miner to access grpc api

### PR DESCRIPTION
**Pull request by pritamghanghas**
**[Issue](https://github.com/NebraLtd/helium-miner-software/issues/427)**

- Link: https://github.com/NebraLtd/helium-miner-software/issues/427
- Summary: gateway-rs listens only on local host. Only way to make diag work at this stage is to use host networking.

**How**
- helium-miner and diagnostics use host networking

**Screenshots**

**References**

**Checklist**

- [ ] Tests added
- [x] Cleaned up commit history (rebase!)
- [ ] Documentation added
- [ ] Thought about variable and method names

### CODE REVIEW

File: .github/workflows/update-miner.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
-name: Update Miner GA (Testnet)
+name: Update Gatewayrs GA (Testnet)
 on:
   repository_dispatch:
-    types: [miner-update]
+    types: [gatewayrs-update]
 jobs:
-  miner-update:
+  gatewayrs-update:
```
Comment (marvinmarnold) [inline .github/workflows/update-miner.yml:6]: Until we land the light-miner work, we don't want to overwrite the existing testnet GA bumping behavior do we?

File: templates/docker-compose.template [hunk 2]
```diff
@@ -31,39 +27,34 @@ services:
...
+    network_mode: host
     cap_add:
       - SYS_RAWIO
     devices:
       - {{I2C_DEVICE}}:{{I2C_DEVICE}}
     restart: on-failure
     environment:
-      - DBUS_SYSTEM_BUS_ADDRESS=unix:path=/session/dbus/session_bus_socket
       - FIRMWARE_VERSION={{FIRMWARE_VERSION}}
+      - I2C_DEVICE={{I2C_DEVICE}}
```
Comment (marvinmarnold) [inline templates/docker-compose.template:49]: Why does this line need to be added now?

Comment (marvinmarnold) [inline templates/docker-compose.template:49]: Why does this line need to be added now?

File: .github/workflows/update-miner.yml (hunk 1 above)
Comment (pritamghanghas) [reply .github/workflows/update-miner.yml:6]: by mistake raised against master. The PR should have been against light-hotspot branch.

**Comment by pritamghanghas** [comment]
by mistake raised against master.

**Pull request closed without merging**